msg/ZFS-8000-ER/index.html

```diff
@@ -225,4 +225,91 @@
 all pools are healthy
 </pre>
 </td></tr></table>
+<br><br>
+<hr>
+<br><br>
+<!-- end: templates/success -->
+<!-- start: article -->
+<!-- code: ZFS-8000-ER -->
+<!-- keys: errata -->
+<table width="80%"><tr><td>
+<a name="4"><b>ZFS Errata #4</b></a>
+<p>
```

Comment:
<3 that this file is raw HTML.  My 90's neurons are coming back to life!

Reply:
haha. yeah this is about the closest that I've gotten to web development at all :)